doc/classes/SplitContainer.xml

```diff
@@ -16,13 +16,40 @@
 				Clamps the [member split_offset] value to not go outside the currently possible minimal and maximum values.
 			</description>
 		</method>
+		<method name="get_drag_area_control">
+			<return type="Control" />
+			<description>
+				Returns the drag area [Control]. For example, you can move a pre-configured button into the drag area [Control] so that it rides along with the split bar. Try setting the [Button] anchors to [code]center[/code] prior to the [code]add_child()[/code] call.
+				[codeblock]
+				var barnacle_button: Button = get_node("%BarnacleButton")
+				remove_child(barnacle_button)
+				get_node("%SplitContainer").get_drag_area_control().add_child(barnacle_button)
```

Comment:
This can be simply
```suggestion
				$BarnacleButton.reparent($SplitContainer.get_drag_area_control())
```
(I used `$`, because it's more familiar)

Reply:
Will do.